component/class/defaults.yml

```diff
@@ -612,3 +612,65 @@ parameters:
 
           # Generic config for objecstorage composite
           secretNamespace: ${crossplane:namespace}
+
+  billing:
+    namespace: appuio-cloud-reporting
+    clusterId: ${cluster:name}
+    salesOrderID: "" #Update once the fact is available in Leutenant
+    prometheus:
+      url: http://prometheus-operated.prometheus-system:9090/prometheus
```

Comment:
I guess this will be changed to mimir later?

Reply:
I had it localhost, we can change later on when we deploy

Reply:
Good, just so it doesn't get missed :)

Also doesn't mimir require any sort of authentication?